mlrun/api/utils/clients/log_collector.py

```diff
@@ -203,12 +204,32 @@ async def has_logs(
         :param raise_on_error: Whether to raise an exception on error
         :return: Whether the log collector service has logs for the given run
         """
+
+        def _retryable_error(error_message: str) -> bool:
+            retryable_error_regex = [
+                # when multiple routines in the log collector service try to search the same directory,
+                # one of them can fail with this error
+                "readdirent.*resource temporarily unavailable"
```

Comment:
take a look at `regex.py` this is the place where we define those regex patterns, I am not sure we want to add this regex to there, but FYI